delay/delay.go

```diff
@@ -222,6 +226,12 @@ func (f *Function) Task(args ...interface{}) (*taskqueue.Task, error) {
 	}, nil
 }
 
+// Request returns the special task-queue HTTP request headers for the current
+// task queue handler.  Must only be called from within a delay.Func.
+func RequestHeaders(c context.Context) *taskqueue.RequestHeaders {
+	return c.Value(headersContextKey).(*taskqueue.RequestHeaders)
```

Comment:
If it's called outside a delay.Func, should it return nil, or panic? At the moment, it panics.